### DISCUSSION TITLE
Download>Library for "User" shows books I have removed from Library.

### DISCUSSION THREAD

**Discussion by Rexadev** [audiamus/BookLibConnect#70] [General]
How to remove them. I have downloaded and moved them to another directory

**Comment by audiamus** [comment]
The download and conversion state is maintained in a local SQLite database. For performance reasons, this database is only updated if you re-select a book that has been downloaded, decrypted and exported before. And in the end, a check-mark or an open lock icon still represents the last state even if you later moved the output elsewhere.
